Bookish/Bookish.DataAccess/LibraryService.cs

```diff
@@ -13,5 +21,40 @@ public IEnumerable<Book> GetBooks()
         {
             return databaseConnection.Query<Book>("SELECT * FROM Books");
         }
+
+        public Book GetCopies(string isbn)
+        {
+            var query =
+                "SELECT LibraryBooks.isbn, Books.title, Books.authors, LibraryBooks.bookId, Users.username, Loans.dueDate " +
+                "FROM LibraryBooks " +
+                "JOIN Books ON LibraryBooks.isbn=Books.isbn " +
+                "LEFT JOIN Loans ON LibraryBooks.bookId=Loans.bookId " +
+                "LEFT JOIN Users ON Users.userId=Loans.userId " +
```

Comment:
Is this your users table or the authentication one? 🙂

Reply:
Currently my Users table as I don't have the borrow feature implemented so wouldn't be able to test the loans page. Will be changing to auth table after implementing ability to borrow

Reply:
Why do you need to wait for that? You can still add entries to your loans table, just use the ids from the auth user one 🙂

Reply:
Feel free to do it as a separate PR to this though 🙂